hammer-db/release.multi-arch.sh

```
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -eu
+
+if ! docker info > /dev/null 2>&1; then
+  echo 'ERROR: docker engine not running. Build failed.' >&2;
+  exit 1;
+fi
+
+unixtime=`date +%s`;
```

Shellcheck (0.10.0)

[warning] unixtime appears unused. Verify use (or export if used externally).

(SC2034)